packages/cli-repl/src/mongosh-repl.spec.ts

```diff
@@ -23,6 +23,11 @@ import MongoshNodeRepl from './mongosh-repl';
 import { parseAnyLogEntry } from '../../shell-api/src/log-entry';
 import stripAnsi from 'strip-ansi';
 
+function nonnull<T>(value: T | null | undefined): NonNullable<T> {
+  if (!value) throw new Error();
+  return value;
+}
```

Comment:
There are some drive-by fixes for type errors in the test files here, mostly things I missed in #1849 because we apply different TS configs for out test and source files